epidose/device/update.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+#
+# Updates software
```

Comment:
Explain when this is executed, under what conditions, how does it work.

Reply:
OK.